### PR TITLE
Fix parameter name in documentation of `fileCheckOutput`.

### PR DESCRIPTION
**Pull request by vguerra**
source -> against.

### CODE REVIEW

File: Sources/FileCheck/FileCheck.swift [hunk 1]
```diff
@@ -110,7 +110,7 @@ extension FileCheckSource: ExpressibleByStringLiteral {
 /// - parameter checkNot: Specifies zero or more prefixes to implicitly reject
 ///   in the output stream.  This can be used to implement LLVM-verifier-like
 ///   checks.
-/// - parameter file: The file to check against.  Defaults to the file that
+/// - parameter against: The source of Check Strings  to check against.  Defaults to the file that
```
Comment (MaxDesiatov) [inline Sources/FileCheck/FileCheck.swift]:
```suggestion
/// - parameter against: The source of Check Strings to check against.  Defaults to the file that
```

Comment (vguerra) [reply Sources/FileCheck/FileCheck.swift]: Thanks for spotting that extra space. I committed your suggestion.

File: Sources/FileCheck/FileCheck.swift [hunk 2]
```diff
@@ -110,7 +110,7 @@ extension FileCheckSource: ExpressibleByStringLiteral {
 /// - parameter checkNot: Specifies zero or more prefixes to implicitly reject
 ///   in the output stream.  This can be used to implement LLVM-verifier-like
 ///   checks.
-/// - parameter file: The file to check against.  Defaults to the file that
+/// - parameter against: The source of Check Strings to check against.  Defaults to the file that
```
Comment (dduan) [inline Sources/FileCheck/FileCheck.swift]: the argument is `source`. `against` is the parameter label.

Comment (vguerra) [reply Sources/FileCheck/FileCheck.swift]: oh yes, indeed. Thanks for the remark, fixed in [7a6517d](https://github.com/llvm-swift/FileCheck/pull/27/commits/7a6517d54dd62fb253fd01ce06c50e72247820b3).

**Review by dduan — APPROVED** [review]

File: Sources/FileCheck/FileCheck.swift (hunk 2 above)
Comment (dduan) [reply Sources/FileCheck/FileCheck.swift]: thanks!

**Pull request merged**